src/index.html

```diff
@@ -7,8 +7,86 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container__main">
+      <a href="#home" class="logo">
+        <img src="images\logo.png" alt="logo">
+      </a>
+
+      <div class="space"></div>
+
+      <nav>
+        <ul class="list">
+          <li class="list__item">
+            <a
+              href="https://www.moyo.ua/ua/portal_brand/apple/"
+              class="nav__link is-active">
```

Comment:
the closing tag should be aligned with opened tag
```suggestion
              class="nav__link is-active"
            >
```

Reply:
Thanks. But I don't know why tests are failed. I`ve wrote to the chat, and got answer: 
![image](https://user-images.githubusercontent.com/126285904/228221410-6e0a31b4-c7cd-4924-bdb4-027f1503355b.png)
![image](https://user-images.githubusercontent.com/126285904/228221534-e2769579-2967-4365-9b39-03ca20dcdd84.png)

I would be very happy if you could help me figure it out